docs/api-reference/deck.md

```diff
@@ -377,6 +377,26 @@ Notes:
 * The query methods offer more flexibility for developers to handle events compared to the built-in hover and click callbacks.
 
 
+## Member Variables
+
+#### metrics
+
+A map of various performance statistics for the last 60 frames of rendering. Metrics gathered in deck.gl are the following:
+- 'fps': average number of frames rendered per second
+- 'updateAttributesTime': time spent updating layer attributes
+- 'layersRedrawn': number of layers drawn
+- 'setPropsTime': time spent setting layer properties
```

Comment:
This tracks the time spent in `deck.setProps` so it should be "deck properties"?